com/copy_ydb_dist_dir.csh

```diff
@@ -0,0 +1,23 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2005-2016 Fidelity National Information		#
```

Comment:
No need of FIS copyrights in a new file that we create. Only YottaDB copyrights needed.